### PR TITLE
doc: make kwok docs better for local development

### PR DESCRIPTION
**Pull request by jwcesign**
Fixes #N/A 

**Description**

doc: make kwok docs better for local development 

**How was this change tested?**

no code change

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -53,7 +53,7 @@ e2etests: ## Run the e2e suite against your local cluster
 
 # Run make install-kwok to install the kwok controller in your cluster first
 # Webhooks are currently not supported in the kwok provider.
-apply: verify build ## Deploy the kwok controller from the current state of your git repository into your ~/.kube/config cluster
```
Comment (jwcesign) [inline Makefile:56]: Separate verify and apply operations, if not, it costs much time to verify when I need to deploy again after the code changed.

**Comment by jwcesign** [comment]
Please take a look: 
cc @ellistarn @jonathan-innis @tzneal @bwagner5 @njtran

File: Makefile (hunk 1 above)
Comment (njtran) [reply Makefile:56]: Why do you want to skip verify? This does some yq logic to add into CRDs before we apply them on the `make apply` target.

**Review by njtran — COMMENTED** [review]
One comment

File: Makefile (hunk 1 above)
Comment (jwcesign) [reply Makefile:56]: Most of the time, I only modify some function logic. Verifying it every time is unnecessary, which makes my development process slow.

This is my original intention for this modification.

Comment (jwcesign) [reply Makefile:56]: If someone wants to develop:
they should run:
```sh
make verify
make apply #Run this command again to redeploy if the code has changed
```

**Comment by jwcesign** [comment]
Please take a look @ellistarn @jonathan-innis @tzneal @bwagner5 @njtran

**Review by njtran — COMMENTED** [review]
I think the docs look good, I'm not convinced on removing the verify part of make apply.

**Comment by jwcesign** [comment]
> I think the docs look good, I'm not convinced on removing the verify part of make apply.

I delete that modification part.

**Review by njtran — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**